index.html

```diff
@@ -191,10 +191,31 @@ <h3 id="About-me">About me</h3>
                 </div>
             </section>
         </div>
-
-
-        
-     </main>
+        <div class="footer">
+            <section id="contact-form" class="work-section">
+              <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in</p>
+              <form action="https://formspree.io/forms">
+                <input type="text" class="font-style name-email" name="fullname" maxlength="30" id="fullname" placeholder="Full name" required>
+                <span class="hr-line form-hr"></span>
+                <input type="email" class="font-style name-email" name="email" id="email" placeholder="Email address" required>
+                <span class="hr-line form-hr"></span>
+                <textarea id="textarea" class="font-style" name="message" maxlength="500" placeholder="Enter text here" cols="30" rows="10" required></textarea>
+              </form>
+              <button class="custom">Get in touch</button>
```

Comment:
- [ ] Kindly add the `button` inside the form and add the attribute `type` with the value `submit` to enable form submission.